source/github-helpers/index.ts

```diff
@@ -177,7 +177,7 @@ export function shouldFeatureRun({
 
 export async function isArchivedRepoAsync(): Promise<boolean> {
 	// Load the bare minimum for `isArchivedRepo` to work
-	await elementReady('#repository-container-header');
+	await elementReady('main > div');
```

Comment:
This should work for both. The first `div` in `main` appears to always be the flash banner itself or some other empty element if the banner is missing.

I can't use `main` because that would wait for the whole page before ever resolving, whether the banner exists or not